### PR TITLE
Gradient Boosting Regressor

### PR DESCRIPTION
**Pull request by abdoulayegk**
### **Describe your change:**

Adding a Gradient boosting regressor algorithm to predict the Boston house price.

### CODE REVIEW

**Comment by spamegg1** [comment]
@abdoulayegk Build log says
> 1     E128 continuation line under-indented for visual indent
4     E225 missing whitespace around operator
26    E231 missing whitespace after ','
5     E251 unexpected spaces around keyword / parameter equals
3     E261 at least two spaces before inline comment
1     E262 inline comment should start with '# '
9     E265 block comment should start with '# '
1     E266 too many leading '#' for block comment
1     E271 multiple spaces after keyword
4     E301 expected 1 blank line, found 0
2     E302 expected 2 blank lines, found 1
2     E402 module level import not at top of file
5     E501 line too long (96 > 88 characters)
2     F401 'sklearn.ensemble.RandomForestRegressor' imported but unused
1     F821 undefined name 'is_empty'
16    W291 trailing whitespace
2     W293 blank line contains whitespace
2     W391 blank line at end of file

Most of these can be easily fixed by running black on your code before submitting: `pip3 install black` then `black filename.py` on the command line.

**Comment by abdoulayegk** [comment]
@spamegg1 Thanks it is helpful

**Comment by abdoulayegk** [comment]
hey can anyone  help me here to fix this I can't figure out The command "scripts/validate_filenames.py" failed and exited with 1 during  meaning.

**Review by poyea — APPROVED** [review]
Thank you for your pull request!🤩
